.github/workflows/add-to-backlog.yml

```
@@ -0,0 +1,20 @@
+name: Add issues to shared backlog
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    branches-ignore: 
+      - "dependabot/**/*"
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
```

Just the action that can be upgraded:

```suggestion
      - uses: actions/add-to-project@v0.5.0
```